dsapi/service/ds_base_service.py

```diff
@@ -46,14 +46,16 @@ def _request(self, path, method='GET', body=None, headers=None):
         """
         url = '{}{}'.format(self._url_base, path)
         headers = self._headers() if headers is None else headers
+        headers["Accept"] = "*/*"
         response, content = super(DSBaseService, self)._request(url,
                                                                 method=method,
                                                                 body=str(body).replace("'", '"'),
                                                                 headers=headers)
         if int(response['status']) == 200:
```

Comment:
Some of the DS API return 204 as well, so I think we should consider 200 to 299 as success.

Reply:
@shikhin-metron Please check if the api calls made with this function are returning which status codes.

Reply:
will add condition to accept for all status codes between 200 and 299. Will that be correct?